plugins/modules/ipaautomember.py

```diff
@@ -0,0 +1,388 @@
+                                         condition_args])
+
+                    for _inclusive in inclusive_del:
+                        key, regex = _inclusive.split("=")
+                        condition_args = gen_condition_args(
+                            grouping, key, inclusiveregex=regex)
+                        commands.append([name, 'automember_remove_condition',
+                                         condition_args])
+
+                if exclusive is not None:
+                    for _exclusive in exclusive_add:
+                        key, regex = _exclusive.split("=")
+                        condition_args = gen_condition_args(
+                            grouping, key, exclusiveregex=regex)
+                        commands.append([name, 'automember_add_condition',
+                                         condition_args])
+
+                    for _exclusive in exclusive_del:
+                        key, regex = _exclusive.split("=")
+                        condition_args = gen_condition_args(
+                            grouping, key, exclusiveregex=regex)
+                        commands.append([name, 'automember_remove_condition',
+                                         condition_args])
+
+            elif state == 'absent':
+                if action == "service":
+                    if res_find is not None:
+                        commands.append([name, 'automember_del',
+                                         {'type': to_text(grouping)}])
+
+                elif action == "member":
+                    if res_find is None:
+                        ansible_module.fail_json(msg="No service '%s'" % name)
+
+                    if inclusive is not None:
+                        for _inclusive in inclusive:
+                            key, regex = _inclusive.split("=")
+                            condition_args = gen_condition_args(
+                                grouping, key, inclusiveregex=regex)
+                            commands.append(
+                                [name, 'automember_remove_condition',
+                                 condition_args])
+
+                    if exclusive is not None:
+                        for _exclusive in exclusive:
+                            key, regex = _exclusive.split("=")
+                            condition_args = gen_condition_args(
+                                grouping, key, exclusiveregex=regex)
+                            commands.append([name,
+                                             'automember_remove_condition',
+                                            condition_args])
+
+            elif state == "rebuild":
+                if res_find is None:
+                    ansible_module.fail_json(msg="No service '%s'" % name)
+                commands.append([None, 'automember_rebuild',
+                                 {"type": to_text(grouping)}])
+
+        errors = []
+        for name, command, args in commands:
```

Comment:
ansible-freeipa modules must support `check_mode`. Check file [utils/templates/ipamodule+member.py.in](https://github.com/rafasgj/ansible-freeipa/blob/masterutils/templates/ipamodule+member.py.in), [line 289](https://github.com/freeipa/ansible-freeipa/blob/0cfd07a70966e5e43975c415be0445a774a1cf45/utils/templates/ipamodule%2Bmember.py.in#L289) to see how this is being implemented.

Reply:
Done